Make Qt statically linked with libgcc_s_seh-1.dll and libstdc++-6.dll.

diff --git a/mkspecs/win32-g++/qmake.conf b/mkspecs/win32-g++/qmake.conf
index 5e9923357f..0c200eaa57 100644
--- a/mkspecs/win32-g++/qmake.conf
+++ b/mkspecs/win32-g++/qmake.conf
@@ -26,4 +26,6 @@ QMAKE_CFLAGS_LTCG       = -flto
 QMAKE_CXXFLAGS_LTCG     = $$QMAKE_CFLAGS_LTCG
 QMAKE_LFLAGS_LTCG       = $$QMAKE_CFLAGS_LTCG
 
+QMAKE_LFLAGS           += -static-libgcc -static-libstdc++
+
 load(qt_config)
